src/commands/init.js

```diff
@@ -80,6 +83,96 @@ async function handleDatabaseConfiguration() {
   return inquirer.prompt(promptContent);
 }
 
+function validateEndpoint(input) {
+  if (!VALIDATION_REGEX_URL.test(input)) {
+    return 'Application input must be a valid url.';
+  }
+  if (!VALIDATION_REGEX_HTTPS.test(input)) {
+    return 'HTTPS protocol is mandatory, except for localhost and 127.0.0.1.';
+  }
+  return true;
+}
+
+function getApplicationPortFromCompleteEndpoint(endpoint) {
+  if (endpoint.startsWith('http://localhost:') || endpoint.startsWith('http://127.0.0.1:')) {
+    return endpoint.substring(17);
+  }
+  return null;
+}
+
+async function getContentToAddInDotenvFile(config) {
+  const authSecret = await generateKey();
+  let contentToAddInDotenvFile = '';
+
+  if (config.applicationPort) {
+    contentToAddInDotenvFile += `APPLICATION_PORT=${config.applicationPort}\n`;
+  }
+  if (config.databaseUrl) {
+    contentToAddInDotenvFile += `DATABASE_URL=${config.databaseUrl}\n`;
+  }
+  if (config.databaseSchema) {
+    contentToAddInDotenvFile += `DATABASE_SCHEMA=${config.databaseSchema}\n`;
+  }
+  if (config.databaseSSL !== undefined) {
+    contentToAddInDotenvFile += `DATABASE_SSL=${config.databaseSSL}\n`;
+  }
+  contentToAddInDotenvFile += `FOREST_AUTH_SECRET=${authSecret}\n`;
+  contentToAddInDotenvFile += `FOREST_ENV_SECRET=${config.forestEnvSecret}`;
+  return contentToAddInDotenvFile;
+}
+
+function commentExistingVariablesInAFile(fileData, config) {
+  let variablesToComment = {
+    'FOREST_AUTH_SECRET=': '# FOREST_AUTH_SECRET=',
+    'FOREST_ENV_SECRET=': '# FOREST_ENV_SECRET=',
+  };
+  if (config.applicationPort) {
+    variablesToComment = {
+      ...variablesToComment,
+      'APPLICATION_PORT=': '# APPLICATION_PORT=',
+    };
```

Comment:
```suggestion
    variablesToComment['APPLICATION_PORT='] = '# APPLICATION_PORT=';
```

Reply:
👍 
I was not sure what was the best way to do it between those 2.
Do you prefer this one because it's more compact?